tests/test_integration.py

```diff
@@ -18,22 +18,31 @@
 
 
 def make_factory():
-    automap = automap_base()
+    # NOTE: Due to the changes made in the production database and the switch
+    # to using views instead of tables, we have attempted to reconstruct the
+    # schema with our test data.  It is an approximation of what is actually
+    # in production, but we are focused more on ensuring that the behavior and
+    # processing of our scripts and business logic are sound as opposed to
+    # mirroring the data exactly as it is at this time.
+    metadata = sa.MetaData(schema='disclosure')
+    automap = automap_base(bind=db.engine, metadata=metadata)
     automap.prepare(db.engine, reflect=True)
 
     class SchedAFactory(SQLAlchemyModelFactory):
         class Meta:
             sqlalchemy_session = db.session
-            model = automap.classes.fec_vsum_sched_a
-        filing_form = '11'
+            model = automap.classes.nml_sched_a
```

Comment:
Ah, I was curious at to how to handle this.  I think I see what's happening here.  Although I'm a little wary about taking out any references to `disclosure.v_sum_and_det_sum_report`, as the tests are basically assuming that table is incidental to the updates.  Also, and we can talk about this, I can't seem to find at what point the rows are inserted into the partitions for `ofec_sched_a_master` and `ofec_sched_b_master`.